{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2024-25030",
      "ASSIGNER": "psirt@us.ibm.com"
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-532"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://www.ibm.com/support/pages/node/7145725",
          "name": "https://www.ibm.com/support/pages/node/7145725",
          "refsource": "",
          "tags": [

          ]
        },
        {
          "url": "https://https://exchange.xforce.ibmcloud.com/vulnerabilities/281677",
          "name": "https://https://exchange.xforce.ibmcloud.com/vulnerabilities/281677",
          "refsource": "",
          "tags": [

          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "IBM Db2 for Linux, UNIX and Windows (includes Db2 Connect Server) 11.1 stores potentially sensitive information in log files that could be read by a local user.  IBM X-Force ID:  281677."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2024-04-03T13:16Z",
  "lastModifiedDate": "2024-04-03T17:24Z"
}